src/fungible_token.rs

```diff
@@ -32,22 +29,8 @@ pub struct FungibleToken {
     pub account_storage_usage: StorageUsage,
 }
 
-impl Default for fungible_token::FungibleToken {
-    fn default() -> Self {
-        Self::new()
-    }
-}
-
+#[cfg(feature = "contract")]
 impl FungibleToken {
-    pub fn new() -> Self {
-        Self {
-            total_supply: 0,
-            total_supply_near: 0,
-            total_supply_eth: 0,
-            account_storage_usage: 0,
-        }
-    }
-
```

Comment:
I think we should keep new as it is basically expected. I would suggest having something like this?

```rust
pub fn new() -> Self {
    FungibleToken::default()
}
```